packages/paste-core/components/popover/src/PopoverContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import {
+  useNonModalDialogPrimitiveState,
+  NonModalDialogPrimitivePopoverInitialState,
+} from '@twilio-paste/non-modal-dialog-primitive';
+import {PopoverContext} from './PopoverContext';
+
+export type PopoverContainerProps = NonModalDialogPrimitivePopoverInitialState;
+
+const PopoverContainer: React.FC<PopoverContainerProps> = ({children, placement, gutter, ...initialState}) => {
+  const popover = useNonModalDialogPrimitiveState({placement, gutter, ...initialState});
```

Comment:
I think there is an option to make a popover a focus trap, I would like to make this mandatory. 

You really shouldn’t be able to tab out of a popover.

Reply:
I'm setting the `modal` option so now focus is trapped.